tests/integration/internal/tests/api/sandbox_list_test.go

```diff
@@ -47,9 +47,13 @@ func TestSandboxList(t *testing.T) {
 func TestSandboxListWithFilter(t *testing.T) {
 	c := setup.GetAPIClient()
 
-	sbx := utils.SetupSandboxWithCleanup(t, c)
+	// standard sandbox
+	_ = utils.SetupSandboxWithCleanup(t, c)
 
-	metadataString := "sandboxType=test"
+	// sandbox with custom metadata
+	sbx := utils.SetupSandboxWithCleanupWithTimeout(t, c, 30, &api.SandboxMetadata{"favouriteColor": "blue"})
```

Comment:
is there a reason you use `WithTimeout`?

Reply:
`SetupSandboxWithCleanup` just wraps `SetupSandboxWithCleanupWithTimeout`. I did not want to expose metadata param directly into `SetupSandboxWithCleanup` because it will cause nill for every use of it.